--- src/src/video/android/SDL_androidwindow.c.orig	2018-12-24 10:37:24.520971234 -0800
+++ src/src/video/android/SDL_androidwindow.c	2018-12-30 12:15:41.946628051 -0800
@@ -117,21 +117,25 @@
         return;
     }
 
-    SDL_WindowData * data = (SDL_WindowData *)window->driverdata;
+	{
+		SDL_WindowData * data = (SDL_WindowData *)window->driverdata;
 
-    if (!data || !data->native_window) {
-        return;
-    }
-
-    int old_w = window->w;
-    int old_h = window->h;
-
-    int new_w = ANativeWindow_getWidth(data->native_window);
-    int new_h = ANativeWindow_getHeight(data->native_window);
-
-    if (old_w != new_w || old_h != new_h) {
-        SDL_SendWindowEvent(window, SDL_WINDOWEVENT_RESIZED, new_w, new_h);
-    }
+		if (!data || !data->native_window) {
+			return;
+		}
+
+		{
+			int old_w = window->w;
+			int old_h = window->h;
+
+			int new_w = ANativeWindow_getWidth(data->native_window);
+			int new_h = ANativeWindow_getHeight(data->native_window);
+
+			if (old_w != new_w || old_h != new_h) {
+				SDL_SendWindowEvent(window, SDL_WINDOWEVENT_RESIZED, new_w, new_h);
+			}
+		}
+	}
 }
 
 void
